themes/theme-b2c-tailwind/src/atoms/link.css

```diff
@@ -0,0 +1,11 @@
+[data-store-link][id='link-as-div'] {
+  @apply w-36 flex flex-col py-1 text-center;
+}
+
+[data-store-link] a {
+  @apply w-7/12 self-center mt-2 rounded border-2 border-green-100 bg-green-100 text-sm;
```

Comment:
How about we leave the default example as a hyperlink and not button style?

![image](https://user-images.githubusercontent.com/3356699/145486893-ace5fd2d-9103-4588-a0ab-53aa39a372b0.png)